src/components/Chat/Chat.js

```diff
@@ -310,7 +311,7 @@ export default class Chat extends Component {
             {token.type === 'emoji' ? (
               <Emoji emoji={token.data} big={bigEmoji} />
             ) : token.type === 'clueref' ? (
-              token.data // for now, don't do anything special to cluerefs
+              this.renderClueRef(token.data)
```

Comment:
interesting that this code already existed! i remember wanting this feature before but i guess never pushed it through